### DISCUSSION TITLE
How do I make profiles?

### DISCUSSION THREAD

**Discussion by charlestbell** [seerge/g-helper#1418] [Q&A] [answered]
Hi, love the app!

I want to have two custom 'modes'. One for plugged-in gaming, one for unplugged web browsing that prioritizes battery life.

I see that I can set a bunch of little options eg fan profile, gpu mode, cpu boost, power limit, undervolting.
But I don't see how to make two profiles that I can easily switch between. 

I only see the default modes of Eco, Balanced and Performance, then a Fan+Power one. 

How do I make custom profiles that I can switch between with just a click or two, without having to changing everylittle thing every time I need different behavior?

System:
Zeph g14 2022, 6800s

**Comment by seerge** [comment, marked as answer]
@charlestbell "profiles" (or modes) are same as in AC. You can customize existing Silent/Balanced/Turbo under ``Fans+Power`` or add a new one (based on currently selected) by pressing ``[+]`` under same ``Fans+Power``

GPU "modes" are not part of the profiles. But Optimized is literally what you need - turns off GPU on battery, and turns on when plugged in. Same for refresh rates (just set it to Auto)

I generally advice to check readme :) it's all covered there

**Reply by charlestbell** [reply]
That clarifies things a lot. Thank you!
